apps/nextjs-website/react-components/components/Editorial-Switch/Editorial-Switch.tsx

```diff
@@ -63,9 +63,9 @@ const EditorialSwitch = ({
         sx={{ display: 'grid', justifyContent: 'center', alignItems: 'center' }}
         background={backgroundColor}
       >
-        <TitleSubtitleBlock
+        <TitleBodyBlock
           title={title}
-          {...(subtitle && { subtitle })}
+          {...(body && { body })}
```

Comment:
```suggestion
          {...(subtitle&& { MarkdownRenderer({ markdown: subtitle, variant: 'body2' }) })}
```
Render subtitle as markdown (add proper variant).